test/verify/check-system-services

```diff
@@ -497,6 +456,81 @@ ExecStart=/bin/awk -f /tmp/mem-hog.awk
         # Check that listen is not shown for .service units
         b.wait_not_present("#listen")
 
+    def testLogs(self):
+        m = self.machine
+        b = self.browser
+
+        self.write_file("/etc/systemd/system/test.service",
```

Comment:
How about factoring this and bin/test-service into a helper method, so that you don't have to repeat it here.

Reply:
done